readthedocs/organizations/models.py

```diff
@@ -128,24 +134,24 @@ def __str__(self):
         return self.name
 
     def get_or_create_stripe_subscription(self):
-        # TODO: remove this once we don't depend on our Subscription models.
-        from readthedocs.subscriptions.models import Subscription
-
-        subscription = Subscription.objects.get_or_create_default_subscription(self)
-        if not subscription:
-            # This only happens during development.
-            log.warning("No default subscription created.")
-            return None
-        return self.get_stripe_subscription()
+        return get_or_create_stripe_subscription(self)
```

Comment:
Can we just remove this method and update the code to use the util function instead? I find having multiple methods/functions with the same name pretty confusing when debugging/exploring the code.